pages/login-contract.rst

```diff
@@ -0,0 +1,79 @@
+#######################
+Login Contracts
+#######################
+
+In the Nuts network, the identity of a user is universally valid. This means the whole network trusts authorities like the BIG, CIBG and BRP instead of the sysadmin of a random care organisation.
+
+When a user wants to access data stored in a different EHR system than they are currently using, they must give their own system consent to request data.
+
+So, we have several parties:
+* A user, probably a care professional.
+* A trusted registry which administres traits about the user such as an identity or medical qualification
+* The users software own software system
+* Another system containing data the user wants to access.
+
+The consent should:
+ * be human readable
+ * be machine readable
+ * be final in time
+ * only be usable by the users software provider
+ * temper proof
+ * versionable
+ * support multiple languages
+ * should fit in the http AUTHENTICATION header
+ * contain the following attributes
+    * The identity of the user
+    * The software system given consent to
+    * The scope of the consent
+    * The purpose of the consent
+    * Period when the consent is valid
+    
+Example
+=======
+
+Login v1:NL Ondergetekende geeft hierbij toestemming aan *SoftwareApplicatie* om uit naam van ondergetekende *data op te vragen* ten behoeven van *het verlenen van goede zorg*. Deze toestemming is geldig tot *2019-02-06T21:00*.
+
+Using regular expressions, a machine can extract all the relevant variables::
+
+  /(version):(language) Ondergetekende geeft hierbij toestemming aan (Acting Party) om uit naam van ondergetekende (Purpose) ten behoeven van (Goal). Deze toestemming is geldig (period)./
+
+
+You can try this simple example over here: https://regex101.com/r/sHfsKq/1
+
+Matching groups
```

Comment:
The why per group is also important to mention: some parts are needed for security, others for matching consent records and the rest for limiting scope.